src/gui/editor/Editor.js

```diff
@@ -96,24 +98,28 @@ export class Editor implements ImageHandler {
 		this._minHeight = minHeight
 		return this
 	}
+	
+	setCreatesLists(createsLists: boolean) : Editor {
+		this._createsLists = createsLists
+		return this
+	}
 
 	initSquire(domElement: HTMLElement) {
 		let squire = new (SquireEditor: any)(domElement,
 			{
 				sanitizeToDOMFragment: this._sanitizer,
 			})
 			.addEventListener('keyup', (e) => {
-				// Resolve logins.getUserController dependenciy and uncomment block
-				// if (!this._tutanotaProperties.sendPlaintextOnly) {
-				if (e.which === 32) {
-					let blocks = []
-					squire.forEachBlock((block) => {
-						blocks.push(block)
-					})
-					createList(blocks, /^1\.\s$/, true) // create an ordered list if a line is started with '1. '
-					createList(blocks, /^\*\s$/, false) // create an ordered list if a line is started with '1. '
+				if (this._createsLists) {
+					if (e.which === 32) {
```

Comment:
I forgot to mention that we have Keys constant and `isKeyPressed()` helper.

Reply:
I went ahead and took care of those. My latest commit is a64ca9977ac27e403e00f3388206f75eb9f332da now.